### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by erictsai-cbi** [kuhe/preact-memo#2]
Such a strange place to house your blog post

* I probably wouldn't even mention optimizely.  That was independent of this and its a pretty short section
* I kind of feel like the removal of Moment can kind of a be a blog post.  We leveraged as much of the native Date API as possible and what really helped with the size was the stripping down of the timezone locales. 
* If there were product gains I would mention them at the end as well.  
* Ryan will probably already mention it but the tone against Angular might be taken as kind of aggressive.  I would maybe start with while we were grateful for the work of the Angular team for where it has gotten us we knew we had to reevaluate our usage of it because of our increasing app size. 
* Title should be SEO friendly. Some suggestions:  "The Journey of Migrating a Large Angular App to Preact" 

Overall great content and thanks for sharing

**Comment by kuhe** [comment]
Yeah, I actually don't know how to format regular word docs anymore, only markdown, but we're editing it now in google docs.

@erictsai-cbi hah, the edited draft is shaping up, and actually in the margin comments I was saying how we need to not call out Angular excessively in deference to all the work that was accomplished with it, and Ryan was saying we can get more views with a title like "Angular is out, Preact is in".

I'll pass your comments onto the other reviewers.

- optimizely: There was a big drop off in PLT after optimizely, more noticeable than the removal of Angular. I have to mention it or I'd be claiming Preact did everything.

- I don't know enough about how the Moment changes went to write that blog post 😕 do you want to write it or get someone on the team who remembers?

- product like conversion? Haven't noticed anything major, but it isn't going down at least

- we're currently on a similar title, `Optimizing for the mobile web: Moving from Angular to Preact` formerly `Optimizing for the mobile web: Angular is out, Preact is in`.
